### DISCUSSION TITLE
PCL文件夹里的文件

### DISCUSSION THREAD

**Discussion by E0x72-21** [Meloong-Git/PCL#5310] [提问] [answered]
PCL文件夹下的这些.dll文件和JavaWrapper.jar是干什么用的，之前的版本好像没有
![image](https://github.com/user-attachments/assets/67e42dea-e5d5-4ec4-88fa-07027c395459)

**Comment by Pigeon0v0** [comment]
你这个文件夹是不是在 %appdata%

**Reply by E0x72-21** [reply]
不是的，这是在D盘下

![image](https://github.com/user-attachments/assets/5abc74a2-ddaf-4c68-8553-4eed3723e44d)

**Comment by MoYuan-CN** [comment, marked as answer]
这是必要的运行库文件和 Java 擦腚纸（[00ll00/java_launch_wrapper](https://github.com/00ll00/java_launch_wrapper)）
默认会释放到缓存文件夹中。如果释放失败，则会释放到 ~/PCL 文件夹中
- 参见 https://github.com/Hex-Dragon/PCL2/issues/4535#issuecomment-2498028398
